assets/panel/debugger.properties

```diff
@@ -386,8 +387,9 @@ editor.removeBreakpoint=Remove breakpoint
 editor.editBreakpoint=Edit breakpoint
 
 # LOCALIZATION NOTE (editor.addConditionalBreakpoint): Editor gutter context
-# menu item for adding a breakpoint condition on a line.
-editor.addConditionalBreakpoint=Add conditional breakpoint
+# menu item for adding/editing a breakpoint condition on a line.
+editor.addConditionalBreakpoint=Add/Edit conditional breakpoint
```

Comment:
Good thing I review strings…

GitHub diff clearly didn't help here, but you changed a string without using a new ID :-\
https://developer.mozilla.org/en-US/docs/Mozilla/Localization/Localization_content_best_practices#Changing_existing_strings

We need a follow up to rename this string ID

Reply:
@lukaszsobek  BTW, is this change expected? Now it describes a completely different function, but there are no code changes associated in this PR.